platform/graphics/layer_opengl.c

```diff
@@ -471,65 +550,24 @@ static void GLDeleteCamera(PLCamera *camera) {
-        if (UseBufferScaling(camera)) {
-            if (camera->viewport.old_r_h != camera->viewport.r_h &&
-                camera->viewport.old_r_w != camera->viewport.r_w) {
-                glDeleteFramebuffers(1, &camera->viewport.buffers[VIEWPORT_FRAMEBUFFER]);
-                glDeleteRenderbuffers(1, &camera->viewport.buffers[VIEWPORT_RENDERBUFFER_DEPTH]);
-                glDeleteRenderbuffers(1, &camera->viewport.buffers[VIEWPORT_RENDERBUFFER_COLOUR]);
-
-                free(camera->viewport.v_buffer);
-                camera->viewport.v_buffer = (uint8_t *) pl_malloc(
-                        (size_t) (camera->viewport.r_w * camera->viewport.r_h * 4));
-                if(camera->viewport.v_buffer != NULL) {
-                    glGenFramebuffers(1, &camera->viewport.buffers[VIEWPORT_FRAMEBUFFER]);
-                    glGenRenderbuffers(1, &camera->viewport.buffers[VIEWPORT_RENDERBUFFER_COLOUR]);
-                    glGenRenderbuffers(1, &camera->viewport.buffers[VIEWPORT_RENDERBUFFER_DEPTH]);
-
-                    /* colour */
-                    glBindRenderbuffer(GL_RENDERBUFFER, camera->viewport.buffers[VIEWPORT_RENDERBUFFER_COLOUR]);
-                    glRenderbufferStorage(GL_RENDERBUFFER, GL_RGB4, camera->viewport.r_w, camera->viewport.r_h);
-                    glBindFramebuffer(GL_DRAW_FRAMEBUFFER, camera->viewport.buffers[VIEWPORT_FRAMEBUFFER]);
-                    glFramebufferRenderbuffer(GL_DRAW_FRAMEBUFFER, GL_COLOR_ATTACHMENT0, GL_RENDERBUFFER,
-                                              camera->viewport.buffers[VIEWPORT_RENDERBUFFER_COLOUR]);
-
-                    /* depth */
-                    glBindRenderbuffer(GL_RENDERBUFFER, camera->viewport.buffers[VIEWPORT_RENDERBUFFER_DEPTH]);
-                    glRenderbufferStorage(GL_RENDERBUFFER, GL_DEPTH_COMPONENT, camera->viewport.r_w, camera->viewport.r_h);
-                    glFramebufferRenderbuffer(GL_DRAW_FRAMEBUFFER, GL_DEPTH_ATTACHMENT, GL_RENDERBUFFER,
-                                              camera->viewport.buffers[VIEWPORT_RENDERBUFFER_DEPTH]);
-
-                    if (glCheckFramebufferStatus(GL_FRAMEBUFFER) != GL_FRAMEBUFFER_COMPLETE) {
-                        GfxLog("invalid framebuffer status on frame!\n");
-                    }
-
-                    ClearBoundBuffers();
-                }
-
-                camera->viewport.old_r_w = camera->viewport.r_w;
-                camera->viewport.old_r_h = camera->viewport.r_h;
-            }
-
-            if(camera->viewport.v_buffer != NULL) {
-                glBindFramebuffer(GL_DRAW_FRAMEBUFFER, camera->viewport.buffers[VIEWPORT_FRAMEBUFFER]);
-            }
-        }
-    }
-
     int w, h;
-    if(UseBufferScaling(camera)) {
-        w = camera->viewport.r_w;
-        h = camera->viewport.r_h;
-    } else {
-        w = camera->viewport.w;
-        h = camera->viewport.h;
+    w = camera->viewport.w;
+    h = camera->viewport.h;
+    if(camera->viewport.autoScale){
```

Comment:
For public API these should be consistent, so use https://en.wikipedia.org/wiki/Snake_case for any public variables.